jumpscale/packages/stellar_stats/bottle/stats_service.py

```diff
@@ -12,10 +12,12 @@
 _ASSET_ISUERS = {
     "test": {
         "TFT": "GA47YZA3PKFUZMPLQ3B5F2E3CJIB57TGGU7SPCQT2WAEYKN766PWIMB3",
+        "FreeTFT": "GBLDUINEFYTF7XEE7YNWA3JQS4K2VD37YU7I2YAE7R5AHZDKQXSS2J6R",
```

Comment:
I think using get_asset can work better than maintaining these constants

Reply:
this requires a wallet, maybe wallet doesn't exist ?